e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep

```diff
@@ -229,7 +229,7 @@ resource iotHub 'Microsoft.Devices/IotHubs@2020-01-01' = {
     eventHubEndpoints: {
       events: {
         retentionTimeInDays: 1
-        partitionCount: 10
+        partitionCount: 4
```

Comment:
I don't recall the exact error, but the build script did not like the event hub partition count being 10